packages/flutter_tools/test/src/fake_process_manager.dart

```diff
@@ -165,7 +165,7 @@ class _FakeProcess implements Process {
 
   @override
   bool kill([io.ProcessSignal signal = io.ProcessSignal.sigterm]) {
-    assert(false, 'Process.kill() should not be used directly in flutter_tools.');
```

Comment:
Why not?

Reply:
This was a bit too opinionated, we kill processes all the time especially if they are servers.

Reply:
sgtm